metrics/import_host_metrics/src/jsonl.rs

```diff
@@ -0,0 +1,88 @@
+use std::io::BufRead as _;
+
+/// A module for reading JSON Lines (jsonl) files.
+#[derive(Debug, Default)]
+pub struct JsonlReader {
```

Comment:
There is a `load_run_summary` function in the summary_model that does something similar to this